fuzz/test-corpus.c

```diff
@@ -40,14 +40,12 @@ static void testfile(const char *pathname)
     FILE *f;
     unsigned char *buf;
     size_t s;
-
-    if (stat(pathname, &st) < 0 || !S_ISREG(st.st_mode))
```

Comment:
Surely this prevents following a symbolic link, which `fopen()` alone would do? i.e. this is a change of behaviour

Reply:
In the fuzz testing, I don't see a problem.  This isn't run by users, at least outside of building and testing OpenSSL.  If an attacker can sneak a symlink in at this time, they've already owned the system.